docs-conceptual/azurermps-5.7.0/install-azurerm-ps.md

```diff
@@ -38,161 +40,137 @@ PowerShellGet 1.6.0   C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1
-of Azure PowerShell are supported. To determine the version of Azure PowerShell you have installed,
-run `Get-Module AzureRM` from your command line.
+Or completely remove Azure PowerShell from your system.
 
 ```powershell
-Get-Module AzureRM -ListAvailable | Select-Object -Property Name,Version,Path
+Uninstall-Module -Name AzureRM -AllVersions
 ```
 
-### Support for classic deployment methods
-
-If you have deployments that use the classic deployment model you can install the Service
-Management version of Azure PowerShell. For more information, see [Install the Azure PowerShell
-Service Management module](/powershell/azure/servicemanagement/install-azure-ps). The Azure and AzureRM modules share
-common dependencies. If you use both the Azure and AzureRM modules, you should install the same
-version of each package.
+To uninstall all versions of Azure PowerShell except for the latest, run the following
+command:
 
-### <a id="update-azps"></a>Updating to a new version of Azure PowerShell
+```powershell
+Get-InstalledModule -Name AzureRM -AllVersions |
+    Where-Object -Property Version -ne $(Get-InstalledModule -Name AzureRM).Version |
+    ForEach-Object {Uninstall-Module -Name $_.Name -RequiredVersion $_.Version}
+```
 
-If you have a previous version of Azure PowerShell installed that includes the Service Management
-module, you may receive the following error:
+## Update the Azure PowerShell module
 
-```Output
-PackageManagement\Install-Package : A command with name 'Get-AzureStorageContainerAcl' is already
-available on this system. This module 'Azure.Storage' may override the existing commands. If you
-still want to install this module 'Azure.Storage', use -AllowClobber parameter.
+You can update your Azure PowerShell installation by running [Update-Module](/powershell/module/powershellget/update-module) to pull the latest available version. This does __not__ uninstall earlier versions.
 
-At C:\Program Files\WindowsPowerShell\Modules\PowerShellGet\1.0.0.1\PSModule.psm1:1772 char:21
-+ ...          $null = PackageManagement\Install-Package @PSBoundParameters
-+                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    + CategoryInfo          : InvalidOperation: (Microsoft.Power....InstallPackage:InstallPackage) [Install-Package], Exception
-    + FullyQualifiedErrorId : CommandAlreadyAvailable,Validate-ModuleCommandAlreadyAvailable,Microsoft.PowerShell.PackageManagement.Cmdlets.InstallPackage
+```powershell
+Update-Module -Name AzureRM
 ```
 
-As the error message states, you need to use the -AllowClobber parameter to install the module. Use
-the following command:
+If you do not want to keep earlier versions, you can run the following command to remove all _but_ the latest version of Azure PowerShell:
 
 ```powershell
-# Install the Azure Resource Manager modules from the PowerShell Gallery
-Install-Module -Name AzureRM -AllowClobber
+Get-InstalledModule -Name AzureRM -AllVersions |
```

Comment:
Again, this does not remove any submodules. So this really isn't getting the intended result. I have a script that will remove the submodules. But uninstall should be a separate article.